<!DOCTYPE HTML>
<!--
	Forty by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Introduction - Subcellular Genetic Populations</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
		<link rel="icon" href="favicon.ico" type="image/x-icon">
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
		<script type="text/javascript" async
  	src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/latest.js?config=TeX-MML-AM_CHTML">
		</script>
		<script type="text/x-mathjax-config">
		MathJax.Hub.Config({
			tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
			});
		</script>
		<script type="text/javascript" async src="path-to-mathjax/MathJax.js?config=TeX-AMS_CHTML"></script>


	</head>
	<body>

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
				<!-- Note: The "styleN" class below should match that of the banner element. -->
					<header id="header" class="alt style1">
						<a href="index.html" class="logo"><strong>Subcellular Genetic Populations</strong> <span>Madeleine Hall</span></a>
						<nav>
							<a href="#menu">Menu</a>
						</nav>
					</header>

					<!-- Menu -->
						<nav id="menu">
							<ul class="links">
								<li><a href="index.html">Home</a></li>
								<li><a href="background.html">Background</a></li>
								<li><a href="model.html">Model</a></li>
								<li><a href="inference.html">Inference</a></li>
								<li><a href="results.html">Results</a></li>
								<li><a href="discussion.html">Discussion</a></li>
								<li><a href="summary.html">Summary</a></li>
								<li><a href="references.html">References & Glossary</a></li>
							</ul>
							<ul class="actions vertical">
								<li><a href="#" class="button special fit">Read More</a></li>

							</ul>
						</nav>

				<!-- Banner -->
				<!-- Note: The "styleN" class below should match that of the header element. -->
					<section id="banner" class="style1">
						<div class="inner">
							<span class="image">
								<img src="images/mitochondria.jpg" alt="" />
							</span>
							<header class="major">
								<h1>Background</h1>
							</header>
							<!-- <div class="content">
								<p>...<br />
								...</p>
							</div> -->

						</div>
					</section>

				<!-- Main -->
					<div id="main">

						<!-- One -->
							<section id="one">
								<div class="inner">
									<header class="major">
										<h2>Mitochondrial DNA</h2>
									</header>
									<p>Mitochondrial DNA (mtDNA) is a closed, double-stranded DNA circle located in mitochondria, where it exists on the scale of hundreds of thousands of copies in an individual. It encodes a number of essential mitochondrial proteins, as well as molecules necessary for intramitochondrial translation. As a result of its unique maternal inheritance pattern and relatively high mutation rate, mtDNA is often used in evolutionary biology and population genetics studies.</p>
								</div>
								<!-- <div>
									<center>


									<img src="images/image.png" alt="" data-position="center center" style="width:330px" />
									<p style="margin-top: 0px">Figure: Map of the human mitochondrial genome. This picture is adapted from work by <a href="https://commons.wikimedia.org/wiki/User:Manudouz" target="_blank">Emmanuel Douzery.</a></p>
								</center>


								</div> -->
							</section>

						<!-- Two -->
							<section id="two" class="spotlights">
								<section>
									<center>
									<div style="width: 520px; margin-left: 50px">



										<img src="images/het_thresh_noback.png" alt="" data-position="center center" style="width:520px" />

										<p style="margin-left: 10px; margin-top: 20px">Figure: Heteroplasmy as a result of replication, and the existence of a biochemical threshold. Figure adapted from <a href="references.html">Stewart and Chinnery, 2015</a>.</p>


									</div>
								</center>


									<div class="content">
										<div class="inner">

											<header class="major">
												<h3>Heteroplasmy</h3>
											</header>
											<p>Heteroplasmy is the presence of more than one type of organellar genome within a cell or individual. Mutations that occur during replication result in heteroplasmy. It can be described by a ratio $m/(w+m)$, with $m$ denoting the number of mutant DNA molecules present, and $w$ the number of wildtype (non-mutant) molecules. In the context of this study, heteroplasmy refers to the presence of more than one form of mtDNA in a cell.



											</p>
											<p>Heteroplasmy is not always harmful, however it has been shown that heteroplasmic mutations of mitochondrial DNA (mtDNA) are an important source of human diseases. A biochemical threshold exists, such that a functional defect occurs only above a certain level of heteroplasmy (e.g. $h = m/(w+m) \geq 0.8$ results in a defect).</p>
											<!-- <ul class="actions">
												<li><a href="generic.html" class="button">Learn more</a></li>
											</ul> -->

										</div>
									</div>
								</section>
								<section>
									<center>
									<div style="width:500px;margin-right: 50px">
										<center>
										<img src="images/mtdna.png" alt="" data-position="center center" style="width: 500px"/>
										<img src="images/mtdna_dis.png" alt="" data-position="center center" style="width: 450px"/>
									</center>
										<p style="margin-right: 10px; margin-left: 10px;margin-top: 20px">Figure: Structure of the human mitochondrial genome, and corresponding connections with several human diseases. This picture is adapted from work by <a href="https://commons.wikimedia.org/wiki/User:Shanel" target="_blank">Shanel Kalicharan</a>.</p>

									</div>
								</center>
									<div class="content">
										<div class="inner">
											<header class="major">
												<h3>Motivation</h3>
											</header>

											<p>mtDNA mutations contribute to human disease across a range of severity. Rare, highly penetrant mutations often affect the nervous system, muscles, heart and endocrine organs, whereas mutations that have a milder contribution can result in common complex traits and late-onset disorders <a href="references.html">(Stewart and Chinnery, 2015)</a>.
											Furthermore, similar mutational signatures have been observed across cancer types, with mtDNA copy number showing variations within and across cancers in correlation with clinical variables <a href="references.html">(Yuan et al., 2017)</a>.
											<!-- <ul class="actions">
												<li><a href="generic.html" class="button">Learn more</a></li>
											</ul> -->
									The ability to quantify heteroplasmy in mtDNA is therefore key in studying age-related disease, cancers, and other health issues. </p>
									<p>Quantification can be relative or absolute. In relative quantification, changes in gene expression are analysed in a given sample relative to another reference sample (such as an untreated control sample). In absolute quantification, no known standards are needed. The target of interest can be directly quantified, with precision determined by the number of replicates of the data in question. The goal of this project is absolute quantification of mtDNA, to obtain explicit quanitites of target analytes in order to investigate heteroplasmy and its effects. .</p>

									<p>Quantitative analysis of mtDNA is challenging in both experimental and computational aspects. Single cell data by nature is concerned with small numbers of molecules. These low copy numbers result in stochasticity, adding complexity to the task of analysis.

									Two different approaches to quantification arise from two types of data: scRNAseq (single-cell RNA sequencing) data, and fluorescence data from qPCR (quantitative PCR).</p>


										</div>
									</div>
								</section>
								<!-- <section>
									<a class="image">
										<img src="images/pic01.jpg" alt="" data-position="25% 25%" />
									</a>
									<div class="content">
										<div class="inner">
											<header class="major">
												<h3>Quantification & Analysis</h3>
											</header>
											<p>Delta delta ct method. DD work.</p>
											<!-- <ul class="actions">
												<li><a href="generic.html" class="button">Learn more</a></li>
											</ul> -->
										<!-- </div>
									</div>
								</section> -->
							</section>

						<!-- Three -->
							<section id="three">
								<div class="inner">
									<header class="major">
										<h2>scRNAseq data</h2>
									</header>
									<p> New technology (first published by <a href ="references.html">Tang et al. in 2009</a>) allows genome-wide transcriptome data to be obtained from single cells, through the use of high-throughput sequencing (scRNAseq). Across a population of cells, the distribution of expression levels for each gene are obtained. The main advantage of scRNAseq over other methods (e.g. bulk RNA-seq or single-cell real-time qPCR) is that the cellular resolution and the genome wide scope makes it possible to address issues that are intractable using other methods.</p>
									<center>
										<img src="images/scRNAseq.jpg" alt="" data-position="center center" style="width: 600px"/>
										<p>Figure: Single cell RNA sequencing workflow (source: <a href="references.html" target="">Kiselev et al., 2018</a>).</p>
									</center>
									<p><a href="references.html">Kiselev et al., 2018</a> outline a scRNAseq pipeline that incorporates computational and statistical methods available. The ideal scRNAseq pipeline involves considering experimental design, processing reads, preparing the expression matrix, and interpreting biological analysis. Experimental design is important as it has implications on the biological analysis that can be carried out downstream. As each sequencing library represents a single cell, significant attention has to be paid to comparison of the results from different cells. Discrepencies are introduced due to low starting amounts of transcripts since the RNA comes from one cell only. It is possible to alleviate these issues through normalization and corrections.</p>
									<p>A main source of discrepancy between the libraries are gene ‘dropouts’, in which a gene is observed at a moderate expression level in one cell but is not detected in another cell <a href="references.html">(Kharchenko, Silberstein, and Scadden 2014)</a>. Dropouts potentially arise because a gene was not expressed in the cell and hence there are no transcripts to sequence. However, dropouts can also be a result of experimental shortcomings: a gene was expressed but transcripts are lost prior to sequencing, or sequencing depth is not sufficient to produce any reads. In the context of mtDNA, as we are unsure of the effects of heteroplasmy on the transcriptome as a whole, this is important to consider.
									One possible solution to this problem is to impute the dropouts in the expression matrix. Two available imputation methods are MAGIC <a href="references.html">(van Dijk et al. 2017)</a> and scImpute <a href="references.html">(Li et al. 2018)</a>. MAGIC imputes missing expression values by sharing information across similar cells, based on the idea of heat diffusion. scImpute determines which values are affected by dropout events based on a mixture model which learns each gene’s dropout probability in each cell.</p>
									<p>The scRNAseq techniques described by <a href="references.html">Kiselev et al.</a> can be applied to mitochondria and mtDNA in order to quantify heteroplasmy. Additionally, in order to study the effects that mtDNA heteroplasmy has on nuclear DNA, transcriptomes of cells with varying heteroplasmy can be compared.
									An alternative approach uses qPCR data for quantification of heteroplasmy in mtDNA, which is the main focus on this project henceforth.</p>
									<!-- <ul class="actions">
										<li><a href="generic.html" class="button next">Get Started</a></li>
									</ul> -->
								</div>
							</section>

							<section id="four">
								<div class="inner">
									<header class="major">
										<h2>qPCR data</h2>
									</header>
									<p> mtDNA in single cells is present in too low a quantity to be measured and analysed directly. qPCR is an experimental technique that amplifies the quantity of a DNA segment, allowing more of the molecule to be available for experimental analysis in addition to absolute inference of copy number. It involves repeatedly heating and cooling DNA segments whilst mixing with free nucleotides. Molecule numbers are progressively amplified, and a fluorescence measurement can be obtained for each cycle. This fluorescence measurement indicates the number of molecules present at that point. </p>
									<center>
										<img src="images/pcr.png" alt="" data-position="center center" style="width: 650px"/>
										<p>Figure: A simple representation of PCR. At each cycle a DNA molecule is denatured, mixed with primers (red) and free nucleotides (yellow) to form two new DNA strands. The process is 'semi-conservative', as each new DNA molecule consists of one new strand and one original strand that served as the template. After $n$ cycles, one DNA molecule yeilds $2^n$ replicates. This picture is adapted from work by <a href="https://www.khanacademy.org/science/biology/biotech-dna-technology/dna-sequencing-pcr-electrophoresis/a/polymerase-chain-reaction-pcr" target="_blank">Khan Academy</a>.</p>
									</center>
								</section>
								<section class="spotlights">
									<section>
										<div style="width:350px;margin-left: 50px;margin-top: 10px">
											<center>
											<img src="images/het_qpcr.png" alt="" data-position="center center" style="width: 350px"/>
											<p>Figure: Cell contents are split into two wells, such that half of the contents can be measured with qPCR detecting only mutant mtDNA, and the other half measured with qPCR detecting all mtDNA molecules. </p>
										</center>
									</div>
									<div data-position="center center" style="margin-top:150px;margin-left:50px;margin-right:20px">
									<p>Heteroplasmy can be quantified by partitioning mtDNA molecules obtained from a single cell into two halves, and performing qPCR on each partition. We can measure the amount of mtDNA present in one partition. In the second partition, we can measure the amount of only the mutated mtDNA present. The ratio of these measurements tells us the level of heteroplasmy.  </p>

								</section>

							</section>
							<section>

										<div class="inner">
											<header class="major">
												<h3>The Standard Curve Method</h3>
											</header>
																				<p>The standard curve method has been used extensively in absolute quantification of parameters from qPCR data. A calibration curve from samples with known properties is constructed. For qPCR data, the standard curve method is known as the comparative $C_T$ method, also referred to as the $2^{-\Delta \Delta C_T}$ or Double Delta Ct method <a href="references.html">(Schmittgen and Livak, 2008)</a>.
																				</p>
																				<p>For each standard in a set of samples where the initial number of molecules $X_0$ is known, the cycle number $C_T$ where the fluorescence exceeds a fixed threshold $T$ is recorded. A calibration curve is constructed from the $(X_0,C_T)$ for each standard. The set of standards is often a dilution series, where the initial copy number is progressively diluted in, for example, factors of 10. This results in a linear calibration curve when plotting $\log(X_)$ against $C_T$. </p>
																				<center>


																					<img src="images/stdcurve.png" alt="" data-position="center center" style="width: 60%"/>
																					<p>Figure: Procedure for constructing of a standard (calibration) curve from a DNA standard. This picture is adapted from work by <a href="http://www.gene-quantification.de/keer-qpcr-book-chapter-7.pdf" target="_blank">Jacquie T. Keer</a>.  </p>
																				</center>
																				<p>This method of constructing a calibration curve, and the standard curve method overall, has major disadvantages: it assumes the amplification efficiency is constant, and does not take into account the stochasticity of qPCR (see <a href="model.html">Model</a>). Therefore, a more sophisticated approach is desirable for accurate estimation of $X_0$.</p>

									<!-- <ul class="actions">
										<li><a href="generic.html" class="button next">Get Started</a></li>
									</ul> -->
								</div>

							</section>


							<section id="next">
								<div class="inner">
									<ul class="actions">
										<center>
										<li><a href="model.html" class="button next">Model</a></li>
									</center>
									</ul>
								</div>
							</section>

					</div>


					<!-- Contact -->
						<!-- <section id="contact">
							<div class="inner">
								<section>
									<form method="post" action="#">
										<div class="field half first">
											<label for="name">Name</label>
											<input type="text" name="name" id="name" />
										</div>
										<div class="field half">
											<label for="email">Email</label>
											<input type="text" name="email" id="email" />
										</div>
										<div class="field">
											<label for="message">Message</label>
											<textarea name="message" id="message" rows="6"></textarea>
										</div>
										<ul class="actions">
											<li><input type="submit" value="Send Message" class="special" /></li>
											<li><input type="reset" value="Clear" /></li>
										</ul>
									</form>
								</section>
								<section class="split">
									<section>
										<div class="contact-method">
											<span class="icon alt fa-envelope"></span>
											<h3>Email</h3>
											<a href="#">madeleine.hall17@imperial.ac.uk</a>
										</div>
									</section> -->
									<!-- <section>
										<div class="contact-method">
											<span class="icon alt fa-phone"></span>
											<h3>Phone</h3>
											<span>(000) 000-0000 x12387</span>
										</div>
									</section> -->
									<!-- <section>
										<div class="contact-method">
											<span class="icon alt fa-home"></span>
											<h3>Address</h3>
											<span>1234 Somewhere Road #5432<br />
											Nashville, TN 00000<br />
											United States of America</span>
										</div>
									</section> -->
								<!-- </section>
							</div>
						</section> -->

					<!-- Footer -->

						<footer id="footer">
							<div class="inner">
							<center>

								<!-- <ul class="icons">
									<li><a href="#" class="icon alt fa-twitter"><span class="label">Twitter</span></a></li>
									<li><a href="#" class="icon alt fa-facebook"><span class="label">Facebook</span></a></li>
									<li><a href="#" class="icon alt fa-instagram"><span class="label">Instagram</span></a></li>
									<li><a href="#" class="icon alt fa-github"><span class="label">GitHub</span></a></li>
									<li><a href="#" class="icon alt fa-linkedin"><span class="label">LinkedIn</span></a></li>
								</ul> -->
								<ul class="copyright">
									<li>&copy; 2017 Madeleine Hall</li><li>Imperial College London</li><li>Design: <a href="https://html5up.net">HTML5 UP</a></li>
									<li> <span class="icon alt fa-envelope"> </span> <a>madeleine.hall17@imperial.ac.uk</a>
								</ul>
							</center>


							</div>
							<ul class="copyright">
							<li style="margin-left: 40px">banner via: <a href="https://goo.gl/kUcYye" target="_blank">https://goo.gl/kUcYye</a></li>

							</ul>
						</footer>


				<!-- </div> -->

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>

	</body>
</html>
